Makefile

```diff
@@ -34,7 +35,6 @@ deb: build
 	-t deb \
 	-v $(VERSION) \
 	--deb-no-default-config-files \
-	--deb-use-file-permissions \
```

Comment:
Hm, without this, are we just getting lucky that the permissions are as we expect? Seems odd to have to remove this. `fpm` lets us specify a uid and gid, no? Could we keep the above but specify `root` for both of those too?